// Copyright 2015 gRPC authors.

syntax = "proto3";

option java_package = "ex.grpc";
option objc_class_prefix = "HSW";

package hellostreamingworld;

service MultiGreeter {

  rpc sayHello (HelloRequest) returns (stream HelloReply) {}
}

message HelloRequest {
  string name = 1;
  string num_greetings = 2;
}

message HelloReply {
  string message = 1;
}
